recipes/Libriheavy/ASR/transformer/train.py

```diff
@@ -0,0 +1,502 @@
+#!/usr/bin/env python3
+"""Recipe for training a Transformer ASR system with Lirbiheavy.
+The system employs an encoder, a decoder, and an attention mechanism
+between them. Decoding is performed with (CTC/Att joint) beamsearch coupled with a neural
+language model.
+
+To run this recipe, do the following:
+> python train.py hparams/conformer_large.yaml
+
+With the default hyperparameters, the system employs a convolutional frontend and a transformer.
+The decoder is based on a Transformer decoder. Beamsearch coupled with a Transformer
+language model is used  on the top of decoder probabilities.
+
+The neural network is trained on both CTC and negative-log likelihood
+targets and sub-word units estimated with Byte Pairwise Encoding (BPE)
+are used as basic recognition tokens.
+
+The experiment file is flexible enough to support a large variety of
+different systems. By properly changing the parameter files, you can try
+different encoders, decoders, tokens (e.g, characters instead of BPE),
+training split (e.g, small, medium, or large), and many
+other possible variations.
+
+
+Authors
+ * Jianyuan Zhong 2020
+ * Mirco Ravanelli 2020
+ * Peter Plantinga 2020
+ * Samuele Cornell 2020, 2021, 2022
+ * Titouan Parcollet 2021, 2022, 2024
```

Comment:
i think you should remove them as they didnt contributed directly to this training recipe for LibriHeavy